### PR TITLE
Fix components

### PR DESCRIPTION
**Pull request by alarcher**
Continuing here to distinguish with gcc-10 fixes since I have now a dedicated build zone.
@AndWac I need to review the few manifest issues you reported earlier.

### CODE REVIEW

**Comment by alarcher** [comment]
@AndWac This batch should be ready.

File: components/x11/urxvt/pkg5 [hunk 1]
```diff
@@ -19,4 +19,4 @@
         "terminal/urxvt"
     ],
     "name": "rxvt-unicode"
```
Comment (AndWac) [inline components/x11/urxvt/pkg5:21]: pkg5 misses the perl update

Comment (alarcher) [reply components/x11/urxvt/pkg5:21]: The file was updated but for some reason it is this perl that is detected: I have run it twice again to check that I did not forget. I suspect something is not updated yet and that makes the dependency produced to the old one.

Comment (AndWac) [reply components/x11/urxvt/pkg5:21]: On my system it is properly detected.

**Review by AndWac — APPROVED** [review]

File: components/x11/urxvt/pkg5 (hunk 1 above)
Comment (alarcher) [reply components/x11/urxvt/pkg5:21]: Maybe I should update the system to a more recent build?

**Comment by AndWac** [comment]
Am 21.07.22 um 21:25 schrieb Aurélien Larcher:
>
> ***@***.**** commented on this pull request.
>
> ------------------------------------------------------------------------
>
> In components/x11/urxvt/pkg5
> <https://github.com/OpenIndiana/oi-userland/pull/8769#discussion_r927031196>:
>
> > @@ -19,4 +19,4 @@
>           "terminal/urxvt"
>       ],
>       "name": "rxvt-unicode"
>
> Maybe I should update the system to a more recent build?
>
>
Definitely.

We seem to have a new problem on the build server. The last three
jenkins runs were successful but the new packages didn't make it into
the repository.
The run times were also quite shorter than the ones before. :(

> DefiniMessage ID:
> ***@***.***>
>

**Pull request merged**